study-group/README.md

```diff
@@ -0,0 +1,119 @@
+| [Intuitive Positioning](#built-in-support)          | ✅ | ✅ | ✅ | ❌ | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ | ✅ |
+| [User Interaction Trigger](#built-in-support)       | ✅ | ❌ | ❌ | ❌ | ❌ | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| [Mentions Accessibility](#policies)                 | ❌ | ✅ | ❌ | ✅ | ✅ | ✅ | ❌ | ✅ | ✅ | ❌ | ❌ | ❌ |
+| [Default Times Out](#policies)                      | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ✅ | ✅ | ✅ | ✅ |
+| [Allows Multiple](#policies)                        | ✅ | ✅ | ✅ | ✅ | ❌ | ❌ | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ |
+| [Allows Duplicates](#policies)                      | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ | ✅ | ❌ | ✅ | ❌ | ❌ | ❌ |
+| [Newest on Top](#policies)                          | ✅ | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| [Close Button Default](#policies)                   | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| [Translucent](#policies)                            | ❌ | ❌ | ❌ | ✅ | ❌ | ❌ | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ |                                                                           
+
+
+
+## Features
+The list of features used in the [Table of Comparison](#table-of-comparison) comes from the major recurring patterns and ideas that showed up in my research of existing toast implementations. This list is by no means exhaustive, but it covers many of the core elements built into popular toast libraries. They approximately break down into three categories: Properties, Built-In Support, and Policies.
+
+### Properties
+Properties of a toast are specific, tangible parts of the object that a developer can set, toggle, or access. These are often the user-facing elements of the toast, and the general consensus of popular libraries regarding these properties is typically what sets toasts apart from other solutions like alerts and dialogs.
+
+- `title`: header text to be displayed above the main message of the toast. 
+
+- `message`: main text contents of the toast. 
+
+- `timeout`: time period (usually in ms) after which the toast disappears.
+
+- Close Button: dedicated button that closes the toast when clicked.
+
+- Call To Action: button(s) displayed on the toast prompting user engagement.
+
+- Callbacks: functions triggered when the toast enters or exits certain states.
+
+- Events: JavaScript events triggered when the toast enters or exits certain states.
+
+- `icon`: pictograph displayed with the toast.
+
+- Progress Bar: element on toast displaying time remaining before timeout.
+
+### Built-In Support
+The Built-In Support establishes how a toast is defined and used, as well as what it's allowed to be used with. It consists of a collection of implementation details, patterns, and technologies that are supported by popular libraries.
+
+- Theme Support: offers built-in theming options, such as `success`, `info`, or `error`.
+
+- Triggered in JS: a toast object is created in JavaScript, then invoked to trigger display to the DOM.
+
+- Component Pattern: a toast HTML component sits in the DOM to be used in the JavaScript.
```

Comment:
This is unclear to me. Do you mean a HTML custom element?

Reply:
It's definitely a bit confusing. 

For 'Triggered in JS'  I was trying to describe that some libraries have you create and define your toast library in your JavaScript (regardless of if you inject it into an existing DOM element). 

For 'Component Pattern' I was trying to describe that some libraries will require you to create an element representing the toast that sits in the DOM, which is either completely configured or just a dummy element to call your toast object from JavaScript on. 

The two aren't necessarily mutually exclusive - you can be 'Triggered in JS' with or without 'Component Pattern, and vice versa. This is rather tricky to explain clearly, I'll change the language of both the explanation and feature which should hopefully make it more clear.

Reply:
Example:

Polymer paper-toast - Triggered in JS ❌ Component Pattern ✅
```html
<paper-button on-click="openToast">Open Toast</paper-button>
<paper-toast id="toast" text="Hello world!"></paper-toast>
```
```js
openToast: function() {
  this.$.toast.open();
}
```

SweetAlert2 - Triggered in JS ✅ Component Pattern ❌
```js
Swal.fire({
  type: 'error',
  title: 'Oops...',
  text: 'Something went wrong!'
});
```

Reply:
Renamed these to 'Configured / Created in JS' and 'Explicit Component Pattern' and clarified language.